packages/stargate/src/signingstargateclient.spec.ts

```diff
@@ -690,6 +690,313 @@ describe("SigningStargateClient", () => {
     });
   });
 
+  describe("signAndBroadcastSync", () => {
+    describe("direct mode", () => {
+      it("works", async () => {
+        pendingWithoutSimapp();
+        const wallet = await DirectSecp256k1HdWallet.fromMnemonic(faucet.mnemonic);
+        const client = await SigningStargateClient.connectWithSigner(
+          simapp.tendermintUrl,
+          wallet,
+          defaultSigningClientOptions,
+        );
+
+        const msg = MsgDelegate.fromPartial({
+          delegatorAddress: faucet.address0,
+          validatorAddress: validator.validatorAddress,
+          amount: coin(1234, "ustake"),
+        });
+        const msgAny: MsgDelegateEncodeObject = {
+          typeUrl: "/cosmos.staking.v1beta1.MsgDelegate",
+          value: msg,
+        };
+        const fee = {
+          amount: coins(2000, "ucosm"),
+          gas: "222000", // 222k
+        };
+        const memo = "Use your power wisely";
+        const transactionHash = await client.signAndBroadcastSync(faucet.address0, [msgAny], fee, memo);
+       
+        expect(transactionHash).toMatch(/^[0-9A-F]{64}$/);
+      });
```

Comment:
Could you end all the new tests with

```ts
await sleep(simapp.blockTime * 1.5); // wait for block inclusion to update account sequence for next test
```

Otherwise the tests ends with a tx in the mempool and the next test gets the old sequence.

Reply:
Seems like this worked!